lib/new_playground.dart

```diff
@@ -1100,3 +1104,81 @@ class TabExpandController {
     _subscriptions.clear();
   }
 }
+
+class NewPadDialog {
+  final MDCDialog _mdcDialog;
+  final Element _dartButton;
+  final Element _flutterButton;
+  final MDCButton _createButton;
+  final MDCButton _cancelButton;
+  final MDCSwitch _htmlSwitch;
+  final DElement _htmlSwitchContainer;
+
+  NewPadDialog()
+      : assert(querySelector('#new-pad-dialog') != null),
+        assert(querySelector('#new-pad-select-dart') != null),
+        assert(querySelector('#new-pad-select-flutter') != null),
+        assert(querySelector('#new-pad-cancel-button') != null),
+        assert(querySelector('#new-pad-create-button') != null),
+        assert(querySelector('#new-pad-html-switch-container') != null),
+        assert(querySelector('#new-pad-html-switch-container .mdc-switch') != null),
+        _mdcDialog = MDCDialog(querySelector('#new-pad-dialog')),
+        _dartButton = querySelector('#new-pad-select-dart'),
+        _flutterButton = querySelector('#new-pad-select-flutter'),
+        _cancelButton = MDCButton(querySelector('#new-pad-cancel-button')),
+        _createButton = MDCButton(querySelector('#new-pad-create-button')),
+        _htmlSwitchContainer = DElement(querySelector('#new-pad-html-switch-container')),
+        _htmlSwitch = MDCSwitch(querySelector('#new-pad-html-switch-container .mdc-switch'));
+
+  Layout get selectedLayout {
+    if (_dartButton.classes.contains('selected')) {
+      if (_htmlSwitch.checked) {
+        return Layout.web;
+      }
+      return Layout.dart;
```

Comment:
super nit: This could be a ternary expression.

(Full disclosure: I mostly wrote this to try out the new multiline comments)